src/pages/terraform/configuration/[[...page]].tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react'
+import terraformData from 'data/terraform.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+
+const basePath = 'configuration'
+const baseName = 'Configuration'
+const product = terraformData as Product
+// TODO: productSlug should possibly actually be `terraform`,
+// TODO: but https://mktg-content-api.vercel.app/api/content/terraform/version-metadata?partial=true
+// TODO: does not return any version with "isLatest: true
+const productSlugForLoader = 'terraform-website'
+
+const additionalComponents = {}
```

Comment:
Same request as other comment about prefixing each line with TODO and same question about `additionalComponents`. 😊